src/docfx/config/TestQuirks.cs

```diff
@@ -14,4 +14,6 @@ internal static class TestQuirks
     public static Func<string, string?>? HttpProxy { get; set; }
 
     public static bool? Verbose { get; set; }
+
+    public static bool? Initializable { get; set; }
```

Comment:
I tend to think this could be a new parameter to `docfx new`.

Reply:
Currently we only use it for tests. In our real usage, there won't be such scenario since all of our contents are located in a repository.

Reply:
Why does implementing this feature require such a flag?